### DISCUSSION TITLE
A Little detail - but helpful

### DISCUSSION THREAD

**Discussion by Bumblebee001** [GrandOrgue/GoOdf#55] [General]
Could I ask for the inclusion of the name of the organ the top (header bar, next to GOODF  x.y.z)? I am often working with two simultaneous instances of GOODF and it helps to see that I'm working on the correct odf.

**Comment by larspalo** [comment]
Should be fixed with 88fd476372a9cfa61461ec8505c276ef8d0dba7f but it's only available when building from the source until a new release eventually is made.

**Reply by Bumblebee001** [reply]
Thanks
